common/changes/@cadl-lang/rest/shared-symbol_2023-02-07-17-20.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/rest",
+      "comment": "Fix issues with multiple copies of compiler core being loaded.",
```

Comment:
My only thought is it might be nice to be a bit more explicit about what you're fixing here.